sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -89,14 +92,44 @@ Follow the instructions in [Creating a service principal using Azure Portal][app
 service principal and a client secret. The corresponding `clientId` and `tenantId` for the service principal can be
 obtained from the [App registration page][app_registration_page].
 
+<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L60-L69 -->
+```java
+ClientSecretCredential credential = new ClientSecretCredentialBuilder()
```

Comment:
iirc, we want to showcase DefaultAzureCredential. I would use that instead of explicitly choosing ClientSecretCredential.

Reply:
Fir now I am removing `ClientSecretCredentialBuilder` sample, and  use `DefaultAzureCredential`. Later if  we want we can add it.